core/src/main/java/org/jivesoftware/spark/ui/login/CertificatesManagerSettingsPanel.java

```diff
@@ -46,7 +47,7 @@
 	private final static Insets DEFAULT_INSETS = new Insets(5, 5, 5, 5);
 	private final LocalPreferences localPreferences;
 	private CertificateController certControll;
-	private JTable certTable = new JTable();
+	private static JTable certTable = new JTable();
```

Comment:
Why did you make this static? It results that all instances of this Panel will share the same instance of the table. I'm not sure if that's desirable.